### PR TITLE
fix running samples script

### PR DESCRIPTION
**Pull request by kristapratico**
1) unblocks running samples with python 2.7
2) continues to run samples even if one fails

### CODE REVIEW

**Review by rakshith91 — APPROVED** [review]

File: scripts/devops_tasks/test_run_samples.py [hunk 1]
```diff
@@ -191,7 +195,7 @@ def run_samples(targeted_package):
         with open(samples_dir_path + "/sample_dev_requirements.txt") as sample_dev_reqs:
             for dep in sample_dev_reqs.readlines():
                 check_call([sys.executable, '-m', 'pip', 'install', dep])
```
Comment (swathipil) [inline scripts/devops_tasks/test_run_samples.py:197]: one of the dependencies for EH is async and can't be installed in python 2.7. would you be able to change it to the following?
```suggestion
                try:
                    check_call([sys.executable, '-m', 'pip', 'install', dep])
                except:
                    pass
```

**Pull request merged**